exercises/concept/lasagna/src/test/java/utils/Lasagna.java

```diff
@@ -0,0 +1,41 @@
+package utils;
+
+public class Lasagna extends ReflectionProxy {
+
+    @Override
+    public String getTargetClassName() {
+        return "Lasagna";
+    }
+
+    public int expectedMinutesInOven() {
+        try {
+            return invokeMethod("expectedMinutesInOven", new Class[]{});
+        } catch (Exception e) {
+            return 0;
```

Comment:
Here you will probably want to throw a UnsupportedOperationException

Reply:
I've seen all the implementations of Reflection Proxies throw UnsupportedOperationException as you mentioned.

Reply:
done